modules/sdk/project-templates/project-templates-api/build.gradle

```diff
@@ -0,0 +1,22 @@
+import aQute.bnd.osgi.Jar;
+
+task createBladeSample {	
+	println rootProject
+	println "this is evaluated"
+	
+	dependsOn ":sdk:project-templates:project-templates:jar"
+	
+	println "this is the project path" project(":sdk:project-templates:project-templates").path
```

Comment:
i couldn't get the api project to see the `project(":sdk:project-templates:project-templates") so i wasnt able to jar it, or get the archivePath.

I think we're doing some sort of wizardry in this gradle project.